cmd/helm/testdata/output/get-release.txt

```diff
@@ -6,6 +6,7 @@ REVISION: 1
 CHART: foo
 VERSION: 0.1.0-beta.1
 APP_VERSION: 1.0
+DESCRIPTION: Release mock
```

Comment:
Adding descriptions should be made in a separate PR

Reply:
Not sure if I understand, this PR removes the flags and enables them by default, so adding the descriptions to the golden files was necessary for the tests to pass

Reply:
ok. please disregard